darwin/dataset/remote_dataset.py

```diff
@@ -715,24 +749,31 @@ def get_report(self, granularity: str = "day") -> str:
         """
 
     @abstractmethod
-    def get_releases(self) -> List["Release"]:
+    def get_releases(self, retry: bool = False) -> List["Release"]:
         """
         Get a sorted list of releases with the most recent first.
 
+        Parameters
+        ----------
+        retry : bool, default: False
+            If True, return  all releases, including those that are not available.
+
         Returns
         -------
         List["Release"]
             Returns a sorted list of available ``Release``\\s with the most recent first.
         """
 
-    def get_release(self, name: str = "latest") -> "Release":
+    def get_release(self, name: str = "latest", retry: bool = True) -> "Release":
```

Comment:
This has to be `True` by default. Otherwise it cannot return releases that are not available for download. We need this to be able to display [this](https://github.com/v7labs/darwin-py/pull/876/files#diff-200952219fbbebac3a477ed4f073d173489e125467b83f5e15ef67105535e274R291-R292) message for pending releases